### ISSUE TITLE
ActivityStreams extensions process

### ISSUE THREAD

**Issue by cwebber** [swicg/general#19]
The SocialCG has been given authority to add extensions to ActivityStreams.  However, we haven't yet defined a process.  We know that we have an [ActivityStreams extension page](https://www.w3.org/wiki/Activity_Streams_extensions), and now we support [versioned contexts](https://www.w3.org/ns/activitystreams-history/), but not much else is nailed down.

* What's the incubation process?
* What is the process to determine that an extension is official?

**Comment by sandhawke** [comment]
One interesting example is the IETF URN Community Registration process, described here: https://tools.ietf.org/html/rfc8141#page-25

Other examples?

I'm thinking something like:

1. Someone makes a draft spec for the term(s).  I'd expect it to be more than one sentence and less than 3 pages.  Maybe we provide a template/outline.  
2. They share it with the group for comments, reasonably addressing all comments. Comments must be constructive & helpful.
3. Once any and all comments are addressed, and things have been calm for a few days, and at least 14 days since the original proposal, it's registered and put in the namespace.

**Comment by strugee** [comment]
@sandhawke to be clear that's independent of any W3C process, correct? So it'd still have to go through the usual publication process for Notes?

**Comment by sandhawke** [comment]
Right.   This doesn't involve any W3C publications or process.   The one thing I think I'd suggest is that submitters make clear IPR statements, so the CG can take over if necessary and someday a WG can put the adopted extensions into a next version of AS.
